### DISCUSSION TITLE
Is there any English NeMo pre-train model based on quartznet15x5 BPE

### DISCUSSION THREAD

**Discussion by mehadi92** [NVIDIA-NeMo/NeMo#5764] [Q&A] [answered]
Or any architecture which has the pre-train model for both BPE and char. I have checked [here](https://catalog.ngc.nvidia.com/orgs/nvidia/collections/nemo_asr). But could not found any

**Comment by titu1994** [comment, marked as answer]
We don't train for both BPE and Char. Most models are trained on BPE because it results in better accuracy and with more stride it is not possible to train CTC models without bpe
